BierBuddy.Core/Visitor.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BierBuddy.Core
+{
+    internal class Visitor
+    {
+        // Basis atributen van de gebruiker.
+        public int ID { get; private set; }
+        public string Name { get; set; }
+        public string Bio { get; set; }
+        public int age { get; private set; }
+
+        // Voorkeuren interesses enz.
+        public List<string> DrinkPreference = new List<string>();
+        public List<string> Interests = new List<string>();
+        public List<string> ActivityPreference = new List<string>();
+
+        // De like, dislike en match atributen.
+        public List<int> Likes = new List<int>();
```

Comment:
gegeven likes of gekregen likes? mis comment of andere naam? geld ook voor dislikes